web-common/package.json

```diff
@@ -40,6 +40,7 @@
     "axios": "^0.27.2",
     "codemirror": "^6.3.2",
     "date-fns": "^2.30.0",
+    "duration-fns": "^3.0.2",
```

Comment:
I am a bit apprehensive on introducing a new library for handling time apart from luxon

Reply:
Good point. I will replace this to use luxon.